examples/Workflow/WorkflowWebApp/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Dapr.Workflow;
+
+// The workflow host is a background service that connects to the sidecar over gRPC
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+// Dapr workflows are registered as part of the service configuration
+builder.Services.AddWorkflow(options =>
+{
+    // Example of registering a "Hello World" workflow function
+    options.RegisterWorkflow<string, string>("HelloWorld", implementation: async (context, input) => 
+    {
+        string result  = "";
```

Comment:
Nit: remove extra space and use `String.Empty`?

Reply:
Just removed this temporary variable that was being overwritten in the next line anyway.